cmd/manager/main.go

```diff
@@ -102,8 +100,13 @@ func main() {
 	}
 	defer r.Unset()
```

Comment:
To be fair, it is very edge case, but it triggered me a little to see a `defer` in the same function with an `os.Exit()`. See https://stackoverflow.com/questions/27629380/how-to-exit-a-go-program-honoring-deferred-calls for a discussion about it.

Reply:
If is fine for you, I will leave it like this for now. 

The whole `Ready` interface usage will go away once we bump the sdk, because is a package they do not longer have. Once we do this update, we can refactor the above to ensure the defer can work with the os.Exit.